src/plugins/data_source_management/public/components/data_source_table/data_source_table.tsx

```diff
@@ -247,18 +239,16 @@ export const DataSourceTable = ({ history }: RouteComponentProps) => {
   };
 
   const handleDisplayToastMessage = ({ id, defaultMessage, color, iconType }: ToastMessageItem) => {
-    if (id && defaultMessage && color && iconType) {
```

Comment:
why do we remove this?

Reply:
Since all four attributes are not optional in `ToastMessageItem`, which means we do not need a if loop condition to verify them.